server-phoenix/lib/helios_web/schema/resolvers/location.ex

```diff
@@ -37,18 +39,20 @@ defmodule HeliosWeb.Schema.Resolvers.Location do
   def moon_phase(parent, _args, _info) do
     phase_ratio =
       rem(
-        DateTime.to_unix(DateTime.now!(parent["time_zone"])) - @first_new_moon_epoch,
+        DateTime.to_unix(DateTime.now!(parent.time_zone)) - @first_new_moon_epoch,
         round(@moon_cycle_time)
       ) / @moon_cycle_time
 
     {:ok, round(phase_ratio * 8) * 0.125}
   end
 
-  def day_announcements(_parent, _args, _info) do
+  def day_announcements(parent, _args, _info) do
+    # {:ok, Repo.all(Announcement |> Announcement.happen_today(parent.time_zone))}
+    {:ok, Repo.all(Announcement |> Announcement.happen_today)}
```

Comment:
This is temporary because happen_today in Announcement temporarily does not take in time_zone.